packages/loot-core/src/server/accounts/parse-file.ts

```diff
@@ -27,10 +34,14 @@ export async function parseFile(
         return parseQIF(filepath);
       case '.csv':
       case '.tsv':
-        return parseCSV(filepath, options);
+        return parseCSV(filepath, options?.hasHeaderRow, options?.delimiter);
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏having an object here would be a bit more scalable. It's possible we'll be adding more and more configuration options here. Having an object will be easier then.